src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1328,6 +1355,33 @@ def _configure_security_policy_storage_params(arg_ctx):
                    arg_group=policy_arg_group,
                    help='The number of days to retain audit logs.')
 
+        c.argument('blob_storage_target_state',
+                   arg_group=policy_arg_group,
+                   help='Indicates whether blob storage is a destination for audit records.',
+                   arg_type=get_enum_type(BlobAuditingPolicyState))
+
+        c.argument('log_analytics_target_state',
+                   arg_group=policy_arg_group,
+                   help='Indicates whether log analytics is a destination for audit records.',
+                   arg_type=get_enum_type(BlobAuditingPolicyState))
+
+        c.argument('log_analytics_workspace_resource_id',
+                   arg_group=policy_arg_group,
+                   help='The workspace ID (resource ID of a Log Analytics workspace) for a Log Analytics workspace to which you would like to send Audit Logs.')
+
+        c.argument('event_hub_target_state',
+                   arg_group=policy_arg_group,
+                   help='Indicates whether event hub is a destination for audit records.',
+                   arg_type=get_enum_type(BlobAuditingPolicyState))
+
+        c.argument('event_hub_authorization_rule_id',
+                   arg_group=policy_arg_group,
+                   help='The resource Id for the event hub authorization rule.')
+
+        c.argument('event_hub_name',
```

Comment:
Arguments should not have `name` in their name. E.g. we have args like `--server` or `--database` as opposed to `--server-name` or `--database-name`

Reply:
fixed